src/dscanner/analysis/label_var_same_name_check.d

```diff
@@ -4,167 +4,189 @@
 
 module dscanner.analysis.label_var_same_name_check;
 
-import dparse.ast;
-import dparse.lexer;
 import dsymbol.scope_ : Scope;
```

Comment:
Why not use the dmd scope? I guess an argument can be mode both sides: (1) we can wither try to replace libdparse and then have a pass to replace `dsymbol` or (2) we can try to do both.

I guess we can just replace libdparse first and then look at replacing dsymbol.

@edi33416 opinions?